crates/starknet-devnet/src/cli.rs

```diff
@@ -130,17 +131,23 @@ pub(crate) struct Args {
     #[arg(help = "Specify the number of the block to fork at;")]
     #[arg(requires = "fork_network")]
     fork_block: Option<u64>,
+
+    #[arg(long = "request-body-size-limit")]
+    #[arg(value_name = "BYTES")]
+    #[arg(help = "Specify the maximum HTTP request body size;")]
+    #[arg(default_value_t = DEVNET_DEFAULT_REQUEST_BODY_SIZE_LIMIT)]
+    request_body_size_limit: usize,
```

Comment:
I'm unsure about using the `usize` type for this, as it is not machine-agnostic.

Reply:
No problem, even if its on 32bit machine 4 bytes are 4_294_967_295 which is 4GB +.

Reply:
True, though I was more interested in having the same behavior on all platforms, but maybe it's not so important